CHANGELOG.md

```diff
@@ -0,0 +1,78 @@
+# Changelog
+
+## DLA-Future 0.4.1
+
+### Bugfixes
+
+- Update project version and export it in CMake. ([#1121](https://github.com/eth-cscs/DLA-Future/pull/1121))
```

Comment:
Ah, I notice I'm using a different format here for the PR link. The plain number will be implicitly linked by GitHub. Do you think that's sufficient for the changelog or should we use the explicit link? I'll update entries either way.

Reply:
I think plain number is sufficient. You need access to github anyway to see the PRs.

Anyone else opinion?

Reply:
Tentatively changed to a plain number in https://github.com/eth-cscs/DLA-Future/pull/1126/commits/d8330b0c8602569ce31b5bb37c4e922d0b31d9f9.